torch/storage.py

```diff
@@ -195,12 +194,35 @@ def complex_float(self):
         """Casts this storage to complex float type"""
         return self._to(torch.cfloat)
 
+    def is_pinned(self, device="cuda"):
+        """
+        Determine whether the CPU storage is already pinned on device.
+
+        :param device (str): String representation of device type to pin memory on.
```

Comment:
You can check the docs build when the CI is finished by with the "Preview C++ docs built from this PR" link in the pytorch-bot comment at the top of your PR. At the moment, the format is a little messed up. Please use this format, like all the other docstrings in pytorch:

```
Args:
    [argument name] ([type]): [argument description]
```

Also, we should support both `str` and `torch.device` types for the `device` arg, since `Tensor.is_pinned` supports both of those types